src/main/java/seedu/address/ui/EventDetailPanel.java

```diff
@@ -0,0 +1,48 @@
+package seedu.address.ui;
+
+import java.time.format.DateTimeFormatter;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
+import seedu.address.model.event.Event;
+
+/**
+ * A UI component that displays detailed information about an {@code Event}.
+ * This panel updates dynamically when a new {@code Event} is provided.
+ */
+public class EventDetailPanel extends UiPart<Region> {
```

Comment:
Can we change the name to `EventDetailView` for consistency with `PersonDetailView`?

Reply:
Sure, I'll change it